deploy/helm/charts/templates/lvm-controller.yaml

```diff
@@ -109,7 +109,7 @@ spec:
               value: controller
             - name: OPENEBS_CSI_ENDPOINT
               value: unix:///var/lib/csi/sockets/pluginproxy/csi.sock
-            - name: LVM_NAMESPACE
+            - name: OPENEBS_NAMESPACE
```

Comment:
This doesn't seem like just a CI change, should be on it's own commit?
Does the code already support this?

Reply:
The code already supports this new name.

Reply:
Should still be in its own commit, this is not a CI change

Reply:
Separated helm and CI change commits.